code/_globalvars/lists/maintenance_loot.dm

```diff
@@ -114,5 +114,6 @@ GLOBAL_LIST_INIT(maintenance_loot, list(
 	/obj/item/throwing_star = 1,
 	/obj/item/circuitboard/machine/allaccess = 2,
 	/obj/item/pen/screwdriver = 8,
+	/obj/item/comb = 10,
```

Comment:
Isn't this extremely common?

Reply:
yes

Reply:
why would maints be filled with more combs than screwdrivers?

Reply:
hair specialists